src/index.html

```diff
@@ -1,8 +1,6 @@
 <!DOCTYPE html>
 <html lang='en'>
   <head>
-    <script crossorigin src="https://unpkg.com/react@18/umd/react.production.min.js"></script>
-    <script crossorigin src="https://unpkg.com/react-dom@18/umd/react-dom.production.min.js"></script>
     <script src='/config.js' type='module'></script>
```

Comment:
Is react being bundled in from `node_modules` now presumably?

Reply:
Yes, indeed, this means _react_ and _react-dom_ will be included there in the bundle, e.g. : 

```
> ls
bundle.0dca980c8297507f0608.js
react-dom.production.min.js
react.production.min.js
```

I'm happy for this solution as it is really nice and _elegant_ ..